app/actions/websocket/teams.ts

```diff
@@ -52,3 +54,62 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: WebSocketMess
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
```

Comment:
does this mean we received an event to add the user to a team on a server we don't have? Should we display an error to the user?

Reply:
This would imply that a team was updated on the server with URL `serverUrl`. The WebsocketManager class contains the list of clients that a user has connected.  

The user should never have a serverDatabase to a serverURL they haven't registered because you would not have a registered client to a ServerURL you did not create. 

@larkox, @enahum could we pass in the validated database to `handleEvent` instead of the serverUrl?  We could then remove all of the `if (!database)` checks in the WS handlers. 

When a handler needs to fetch from the server API, it would need the `serverURL` that can be retreived by the database manager.

Reply:
I think it is not a bad idea. That way we only have one single `if (!database) ...` piece of code, and will reduce the number of lines of the websocket actions. At the end of the day, the websocket actions should only be called at the websocket handler level, so it should be fine.

Maybe keep it like this in this PR and create a ticket to refactor the whole websocket package?

Reply:
> ticket created.  https://mattermost.atlassian.net/browse/MM-41517